### DISCUSSION TITLE
M1 mac support

### DISCUSSION THREAD

**Discussion by deveshsnair** [olive-editor/olive#1840] [Q&A] [answered]
Will Olive be supported for m1 macs?
<img width="328" alt="Screenshot 2022-01-23 at 11 28 22 AM" src="https://user-images.githubusercontent.com/38416612/150666816-1acab63b-df9a-4223-b79d-69290c294b46.png">
<img width="1280" alt="Screenshot 2022-01-23 at 11 28 36 AM" src="https://user-images.githubusercontent.com/38416612/150666886-21a30b3d-96f8-4b0c-8e4a-0fdba48205d0.png">

**Comment by itsmattkc** [comment, marked as answer]
What your screenshots are showing is not an M1 issue, that's just because Olive isn't signed by a developer certificate so macOS is being (overly) cautious and preventing you from opening it. If you right click and select open, it'll give you the option to actually run it.

**Comment by deveshsnair** [comment]
Ok ty for the tip

DEVESH S NAIR
